### PR TITLE
CLI command parsing

### LINKED ISSUE #91: CLI command parsing
You can read more about this in https://github.com/TestArmada/flank/issues/89

Requirements:
 - Add https://pholser.github.io/jopt-simple to parse commands
 - new CLI should work like this:
```java -jar Flank.jar -a <app-apk> -t <test-apk> -c <config-path> -p [package-name]```

`a`, `t` are required, `c` and `p` are optional.

### PR DESCRIPTION
**Pull request by Macarse**
Fixes #91

### CODE REVIEW

File: Flank/src/main/java/com/walmart/otto/Flank.java [hunk 1]
```diff
@@ -113,6 +112,24 @@ public static void main(String[] args) {
     }
   }
 
+  static OptionParser getOptionParser() {
+    return new OptionParser() {
```
Comment (rciovati) [inline Flank/src/main/java/com/walmart/otto/Flank.java]: should we declare `p` as well?

Comment (Macarse) [reply Flank/src/main/java/com/walmart/otto/Flank.java]: oh, great catch. I will fix asap.
I don't use -p so when I run this version with my env I didn't find it. My bad.

Comment (Macarse) [reply Flank/src/main/java/com/walmart/otto/Flank.java]: I just fixed it. Thanks again.

Comment (rciovati) [reply Flank/src/main/java/com/walmart/otto/Flank.java]: Cool, let's wait for @renas then :)

**Comment by renas** [comment]
Thanks @Macarse. Looks good!

**Pull request merged**